deeds-dapp-webapp/src/main/webapp/vue-app/dapp/commonRenting/components/button/RentButtons.vue

```diff
@@ -205,7 +206,7 @@ export default {
     },
     rentalOfferLink() {
       return this.showSeeRentPart
-        && `${this.parentLocation}/marketplace?offer=${this.rentalOffer.id}`;
+        && `${this.parentLocation}/${this.marketplaceLabel}?offer=${this.rentalOffer.id}`;
```

Comment:
I think it should be suffixed by `Url` instead of `Label` ?
```suggestion
        && `${this.parentLocation}/${this.marketplaceUrl}?offer=${this.rentalOffer.id}`;
```

Reply:
yes, the url must be used
done!